prow/jobs/busola/busola-web/periodic-performance-tests-web.yaml

```diff
@@ -0,0 +1,15 @@
+# Periodic jobs
+
+# Description- This prow job is for running performance Tests on kyma web application. 
+periodics:
+- interval: 24h
+  agent: kubernetes
+  name: jmeter
+  spec:
+    containers:
+    - image: i331641/kyma-perf:v1
```

Comment:
Is there a source code for this Dockerfile? We aim to not use docker hub in Prow because we aim to have full control over all images we use in here. Additionally DockerHub has some nasty rate limits that we want to rather not hit.

Reply:
Hi Please find the Dockerfile in this repo- https://github.tools.sap/engsrvperformance/Jmeter-Selenium-Docker/blob/main/Dockerfile

Reply:
@PoojaGidaveer I would suggest you adding this image to https://github.com/kyma-project/test-infra/blob/main/development/image-syncer/external-images.yaml so we can clone it to our GCR. Then use this image here. We don't support using DockerHub directly on Prow.

Reply:
Done.. Please approve the PR created - https://github.com/kyma-project/test-infra/pull/6384

Reply:
@PoojaGidaveer you should change image url to GCR one, example from other job definition: https://github.com/kyma-project/test-infra/blob/main/prow/jobs/test-infra/branchprotector.yaml#L17

Reply:
edited.. Kindly check..